### PR TITLE
Features

### PR DESCRIPTION
**Pull request by lily4178993**
# Awesome Books: Final

This PR is raised to implement the following features according to the project [requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/books/books_with_es6.md).  The goal is to make the code more organized by using modules and also practice the ES6 syntax that I have learned.

## Features implemented 👌
The following features have been implemented:

- [x] A Navigation bar with nav menu items.
- [x] When  any of the links, are clicked the main area changes to display only one of the three content sections.
- [x] Used only plain JavaScript and CSS.

**Added Three content sections:**
- [x] Books list.
- [x] Add book form.
- [x] Contact info.

In addition, the following functionalities from the first [implementation](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects_v1_1.md) have been added:
*   When a user clicks the "Remove" button: The correct book is removed from the collection.
*   Data is preserved in the browser's memory by using local storage.
* When a user fills in the form title and author and presses the add button, a book is added to the list.

## Project requirements
The following requirements have also been fulfilled:
**HTML & CSS:**
* Used semantic HTML tags.
* Used CSS selectors correctly.
* Followed the list of [best practices for HTML & CSS](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md).

**Javascript:**
* Used JavaScript to manipulate DOM elements.
* Used JavaScript events.
* Used objects to store and access data.
* Create a separate file for each module in the modules directory.
* Imported modules into the index.js file.
* Refactored all functions to arrow functions.
* Followed the list of [best practices for JavaScript](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/javascript_best_practices.md).

**Linters:**
* There are [no linter errors](https://github.com/microverseinc/linters-config).

**Documentation:**
* Used correct [GitHub flow](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/git-github/articles/github_flow.md).
* Documented our work [in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md).

  
## How should it be tested ? 🧪 

Clone the repository
```
git clone https://github.com/lily4178993/awesome-books-2
```
Navigate to the folder
```
cd awesome-books-2
```
Checkout the branch
```
git checkout features
```

**Awesome books** is hosted on GitHub pages, so if you want to check this project online, please 😊 [Click here](https://lily4178993.github.io/awesome-books-2/)

_**NOTA BENE: 💀 Unable to fix webhint error!**_

Please kindly review my implementation and suggest any changes and improvements.

I look forward to your feedback.
Thanks for reviewing my PR 😊

### CODE REVIEW

File: modules/dateTime.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+/* eslint-disable import/prefer-default-export */
```
Comment (mwanawabangona) [inline modules/dateTime.js]: - [ ] `required`
- [ ] amazing job so far 👏 
- [ ] Please enable the disabled linters cause it defeats the purpose of installing linter 🥂

File: modules/menu.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
+// decare variable to store menu links
+const menulinks = document.querySelectorAll('.row-list li a');
+
+// eslint-disable-next-line import/prefer-default-export
```
Comment (mwanawabangona) [inline modules/menu.js]: - [ ] `required`
- [ ] amazing job so far 👏 
- [ ] Please enable the disabled linter as well 🥂

**Review by mwanawabangona — CHANGES_REQUESTED** [review]
Hi @lily4178993 ,

Good job so far!

There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
* ✅  No linter errors :heavy_check_mark:
* ✅  Repo looks professional :briefcase:
* ✅  Good quality commits


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: modules/menu.js (hunk 2 above)
Comment (lily4178993) [reply modules/menu.js]: Thank you, @mwanawabangona , for taking the time to review my pull request.

**Review by LionRouge1 — APPROVED** [review]
Hi @lily4178993, 

# STATUS: APPROVED! 🥇 💯 
Your project is complete! There is nothing else to say other than... it's time to merge it :shipit:

done

Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Comment by lily4178993** [comment]
Thank you, @LionRouge1, for taking the time to review my pull request.

**Pull request merged**